slice-core-api/src/main/java/com/cognifide/slice/api/packageinfo

```diff
@@ -0,0 +1 @@
+version 4.2
```

Comment:
Wouldn't be better to assign packageinfo version per package instead of one single for whole API?